README.txt

```diff
@@ -1,21 +1,24 @@
-Methodocracy
+Methodocracy (noun):
+A cultural-political ideology which proposes that optimal decision-making, from the individual to national level, is necessarily based upon the quantity and quality of data, interpretation, and conclusions gathered through tested and reliable methodologies.
 
-"The Super-Power of Problem-Solving"
 
+Hurdles: 
+1. It is prohibitively expensive and time-consuming for the average citizen to access information due to the high cost of scientic journals/articles and the lack of a centralized database.
 
-A free website that facilitates problem-solving. Kind of like Wikipedia, but instead of logging already known information, it focuses on accumulating new knowledge using established methods, especially the scientific method. Credentialed users and average Joe's can use the site alike.
```

Comment:
Ok. I thought that this document was supposed to be a short introduction to the people who visit github. I think the shorter elevator pitch should be in a smaller pamphlet/business card or something.